CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing
+
+## Getting Started
+
+General information about setting up your Python environment, testing modules,
+Ansible coding styles, and more can be found in the [Ansible Community Guide](
+https://docs.ansible.com/ansible/latest/community/index.html).
+
+Information about boto library usage, module utils, testing, and more can be
+found in the [AWS Guidelines](https://docs.ansible.com/ansible/devel/dev_guide/platforms/aws_guidelines.html)
+documentation.
+
+## AWS Collections
+
+There are two related collections containing AWS content (modules and plugins).
+
+### amazon.aws
+This collection contains the `module_utils` (shared libraries) used by both collections.
+Content in this collection is included downstream in Red Hat Ansible Automation Platform.
+
+Code standards, test coverage, and other supportability criteria may be higher in this collection. 
+
+### community.aws
+This collection contains modules and plugins contributed and maintained by the Ansible AWS
+community.  The `community.aws` collection is tested and generally assured to work in
+conjunction with `amazon.aws`.
+
+New modules and plugins developed by the community should be proposed to `community.aws`.
+Content in this collection that is stable and meets other acceptance criteria has the potential
+to be promoted and migrated into `amazon.aws`.
+
+## Submitting Issues
+All software has bugs, and the `amazon.aws` collection is no exception. When you find a bug, 
+you can help tremendously by [telling us about it](https://github.com/ansible-collections/amazon.aws/issues/new/choose).
+
+If you should discover that the bug you’re trying to file already exists in an issue, 
+you can help by verifying the behavior of the reported bug with a comment in that 
+issue, or by reporting any additional information
+
+## Pull Requests
+
+All modules marked stableinterface MUST have integration tests for new features. Upgrading to boto3 shall be considered a feature request.
+All new modules MUST have integration tests.
+Modules marked preview SHOULD have integration tests for new features.
+Bug fixes for modules that currently have integration tests SHOULD have tests added.
+
+Expected test criteria:
+* Resource creation
+* Resource creation again (idempotency)
+* Resource creation under check mode
+* Resource modification
+* Resource modification again (idempotency)
+* Resource modification under check mode
+* Resource deletion
+* Resource deletion under check mode
+
+
+## Communicating
+
```

Comment:
Honestly I was working out the layout by referencing a bunch of other projects, and I can't even recall where I got this Communicating > CoC,IRC structure from.  Do you know if we're devising any sort of common template for collections?  It's been a while since I last looked but none of the collections I checked had started on one afaict.

Reply:
Sadly I don't have a quick or easy answer for you. Most of the collections I'm dealing with will point to the results of this PR from their README files - https://github.com/ansible/ansible/pull/69202/files#diff-5dbb7d345aadad3483b1b86e0eb42ba4R232

So for these "ansible-maintained' collections, the README.md is listing the IRC channels under a 'contributing to this collection' section. That's where a link will be added to the above doc once merged and published.  You can see an example of the README.md here -https://github.com/ansible-collections/arista.eos

All that said, there is a collection template somewheres and yes, we should add a CONTRIBUTING.md to it...and I may just steal the results of this PR to help create that :-)